modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java

```diff
@@ -25,7 +25,7 @@
 import java.util.stream.Collectors;
 
 /**
- * <p>Container for Atlassian Labs Localstack, 'A fully functional local AWS cloud stack'.</p>
+ * <p>Container for Localstack, 'A fully functional local AWS cloud stack'.</p>
```

Comment:
```suggestion
 * <p>Container for LocalStack, 'A fully functional local AWS cloud stack'.</p>
```

Seems we are a bit inconsistent with the naming in our docs, so let's fix it as part of this PR 🙂